proto/cluster.proto

```diff
@@ -152,6 +152,12 @@ message Cluster {
 
   // Output. The time that the cluster deleted.
   google.protobuf.Timestamp deleted_at = 8;
+
+  // Output. The service endpoint of the cluster
+  map<string, string> service_endpoint = 9;
```

Comment:
just curious why make the port a string instead of int32?

Reply:
Sound good, I remember at first we do not have any intend to use `string` or `int32`, and if we would like to use it to represent `nodeIP:port` as a endpoints, I think string would be better. WDYT @Jeffwan

Reply:
This is our internal representation. Honestly, I am not sure whether community like it or not? Any suggestions? Do we need the port or endpoint here?

Reply:
It's a little confusing from an outsider's perspective. 

Would we consider also expose the head node ip here, which is more intuitive?